### PR TITLE
Libical 2.0

### PR DESCRIPTION
**Pull request by Gottox**

### CODE REVIEW

**Comment by Vaelatern** [comment]
SONAME bumps missed for `common/shlibs`

**Comment by Gottox** [comment]
fixed.

File: srcpkgs/libical/template [hunk 1]
```diff
@@ -1,15 +1,15 @@
-# Template build file for 'libical'.
+# Template build file for 'libical'
 pkgname=libical
-version=1.0
+version=2.0.0
 revision=1
 build_style=cmake
 hostmakedepends="cmake perl"
```
Comment (Vaelatern) [inline srcpkgs/libical/template]: `cmake` redundant here

**Comment by Vaelatern** [comment]
Nothing I can see as wrong that should prevent a merge.

**Comment by Vaelatern** [comment]
Correction. `bluez-5.41_1: broken, unresolvable shlib 'libicalvcal.so.1'`

**Comment by ebfe** [comment]
ea8c9a610195e9ac410f754aca2523af995538e0

**Pull request closed without merging**